__tests__/integration/services/applications.js

```diff
@@ -1,12 +1,118 @@
 const applicationService = require('../../../src/services/applications')
+const { Applications } = require('../../../src/database')
+
+const APP_ID = 'create_service_1'
 
 describe('Application service', () => {
-  describe.skip('create',() => {
-    it('creating with valid payload', async () => {
-      const response = await applicationService.create()
+  beforeEach(() => {
+    return Applications.delete(APP_ID).catch()
+  })
+
+  describe('create',() => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
  describe("create",() => {
```